scripts/analyze_ccache_usage.py

```diff
@@ -28,7 +28,8 @@
 
 if len(files) == 0:
   print("No *.ccache-log files found. Did you compile with ccache and the environment variable CCACHE_DEBUG=1?")
-  sys.exit(1)
+  # ToDo(anyone): change back to `sys.exit(1)` as soon as our ccache problem is resolved
```

Comment:
@mrks I added that because otherwise the CI fails. We could also remove the call of the analyzer script from the Jenkinsfile as a hot fix. Any preference?

Reply:
I found a way to disable ccache without changing any file in the Hyrise repo. Could you please remove it again?

Reply:
Done.